build.xml

```diff
@@ -13,7 +13,6 @@
     <property name="build.dependencies" location="dependencies" />
 
     <path id="classpath">
-        <fileset dir="/usr/share/java" includes="**/*.jar" />
```

Comment:
yes as I remember, but I don't remember the details :) may be because of custom java installation